### PR TITLE
Ledger n3

### PR DESCRIPTION
**Pull request by ixje**
**What current issue(s) from Trello/Github does this address?**
https://github.com/CityOfZion/neon-wallet/issues/2147

**What problem does this PR solve?**
not having n3 ledger support

**How did you solve this problem?**
add support

**How did you make sure your solution works?**
I haven't yet. Only parts are tested

**Are there any special changes in the code that we should be aware of?**
no

**Is there anything else we should know?**

- [ ] Unit tests written?

### CODE REVIEW

**Comment by comountainclimber** [comment]
The build is failing because `package.json` was updated _but_ the lock file was not updated... If you run `yarn` it should automatically update the lock file and fix the build

File: app/containers/LoginLedgerNanoS/LoginLedgerNanoS.jsx [hunk 1]
```diff
@@ -278,7 +281,11 @@ export default class LoginLedgerNanoS extends React.Component<Props, State> {
             {this.getStatusIcon(OPEN_APP)}
           </div>
           <div className={styles.ledgerStageText}>
-            <FormattedHTMLMessage id="auth.ledger.navigateToNeoApp" />
+            {chain === 'neo2' ? (
+              <FormattedHTMLMessage id="auth.ledger.navigateToNeoApp" />
+            ) : (
+              <FormattedHTMLMessage id="auth.ledger.navigateToNeo3App" />
```
Comment (comountainclimber) [inline app/containers/LoginLedgerNanoS/LoginLedgerNanoS.jsx]: Can we just use the string `'Navigate to the <strong>NEO3 app</strong> on your device'` here - reason being is that without the other translation files updated users will just see `auth.ledger.navigateToNeo3App` when not using English

**Comment by ixje** [comment]
@comountainclimber Known issue(s):
* open Ledger tab in N2 mode, open the app such that we get an address, now switch to N3 mode. The data will not be refreshed. Same happens if you start with N3 and go back to N2.
* open ledger tab in N3 mode, it will say "Connect and unlock your Ledger device", do that. It will stay on this message until you actually do the second step (open the N3 app). I don't know where to look for the trigger to go to the next step.

**Comment by comountainclimber** [comment]
If you revert these changes then the snapshot tests will pass => https://github.com/CityOfZion/neon-wallet/pull/2215/files#diff-5dcaf0cf2e58b16f97b0d3730d6174b58883b456eaf922f936843ffa5c0e8b93R16

I am still trying to figure out what the other issue breaking the test suite is

**Comment by ixje** [comment]
champ!

**Review by comountainclimber — APPROVED** [review]

File: app/containers/LoginLedgerNanoS/LoginLedgerNanoS.jsx [hunk 2]
```diff
@@ -278,7 +281,7 @@ export default class LoginLedgerNanoS extends React.Component<Props, State> {
             {this.getStatusIcon(OPEN_APP)}
           </div>
           <div className={styles.ledgerStageText}>
-            <FormattedHTMLMessage id="auth.ledger.navigateToNeoApp" />
+            Navigate to the <strong>NEO3 app</strong> on your device
```
Comment (comountainclimber) [inline app/containers/LoginLedgerNanoS/LoginLedgerNanoS.jsx:284]: Now this component always says NEO3 app 😞

Comment (comountainclimber) [reply app/containers/LoginLedgerNanoS/LoginLedgerNanoS.jsx:284]: Fixed with https://github.com/CityOfZion/neon-wallet/commit/e2a3773d150ffd8fff635fb38c4a17f77e334d47

**Pull request merged**